src/hooks/useDetails.js

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react';
+
+export default function useDetails(url) {
```

Comment:
This hook seems to be very general, maybe a more general name would be useful. 🤷‍♀️

Reply:
I need this hook also for my next story to get the Details from a movie. Thats why i named this useSeries :)